presto-pinot/presto-pinot-plugin/pom.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We only need presto-pinot/pom.xml right? These two modules presto-pinot-plugin and presto-pinto-lib are not necessary. They can be merged into presto-pinot.

Reply:
We have a use case for keeping these two separate at Uber: Let me explain:

We have a plugin called presto-rta that exposes two realtime engines AresDB and Pinot as a single RTA connector. There is a metastore that maps tables to one or the other engine. The user does not know which engine the data is actually in. 

In that presto-rta connector, we depend on presto-pinot-lib and presto-aresdb-lib respectively. We can't have a presto plugin yet depend on other plugins because each plugin is in its own classloader. 

So the alternative is to expose the "interesting bits" as vanilla library jars that the other plugin can make use of.

This is similar to how presto has presto-hadoop-hive2 that depends on presto-hive, presto-parquet and presto-orc, where the latter three are non plugin modules. 

We could have also created top level modules for presto-pinot-lib and presto-pinot-plugin but it made more sense to me to keep these modules together as child modules of presto-pinot.

Reply:
Got it. It is similar to `presto-hive` and `presto-hive-hadoop2`.
So instead of making the layout like `presto-pinot/presto-pinot-lib` and `presto-pinot/presto-pinot-plugin`, could you just make two top level modules?
- `presto-pinot-connector` (this is the connector without plugin)
- `presto-pinot` (this is the plugin)

Although I don't quite like the naming, @wenleix, @arhimondr do you have better suggestions?